package.json

```diff
@@ -4,7 +4,8 @@
   "version": "0.1.0",
   "description": "terra-clinical",
   "engines": {
-    "node": "^10.0.0 || ^12.0.0 || ^14.0.0"
+    "node": "10 || 12 || 14",
```

Comment:
Does this do an exact version check or supports all minor or patch versions under them. Should this be written like this `^10 || ^12 || ^14`

Reply:
This supports all minor & patch version:
 https://docs.npmjs.com/about-semantic-versioning

I also double checked the syntax using https://semver.npmjs.com.